versioned_docs/version-1.15/using-dev-envs.mdx

```diff
@@ -150,7 +150,7 @@ $ nodemon server.js
 Server running on port 8080.
 ```
 
-Now, you should be able to view your application when you visit these endpoints. Behind the scenes, the Development Environment in Okteto will synchronize your changes every time you save them locally.
+You should be able to view your application when you visit this endpoint. Behind the scenes, the Development Environment in Okteto will synchronize your changes every time you save them locally.
```

Comment:
There's only one endpoint created so I find that using the plural is a bit confusing.